### PR TITLE
Fix: Go back to KeyOverview after add a Wallet (WalletDetails)

### PR DESCRIPTION
**Pull request by cmgustavo**
https://user-images.githubusercontent.com/237435/203146566-797c20f5-c861-4386-8796-92deec628f91.mov

### CODE REVIEW

File: src/navigation/wallet/screens/AddWallet.tsx [hunk 1]
```diff
@@ -498,10 +498,24 @@ const AddWallet: React.FC<AddWalletScreenProps> = ({navigation, route}) => {
       const wallet = await _addWallet(_associatedWallet, walletName);
 
       if (!withinReceiveSettings) {
-        navigation.navigate('WalletDetails', {
-          walletId: wallet.id,
-          key,
-          skipInitializeHistory: false, // new wallet might have transactions
+        navigation.reset({
```
Comment (rastajpa) [inline src/navigation/wallet/screens/AddWallet.tsx:501]: apparently navigation.reset(...) is deprecated in the latest versions of react native. You should use CommonaAtions.reset(...) instead. Here is an example:
```navigation.dispatch(
                CommonActions.reset({
                  index: 1,
                  routes: [
                    {
                      name: 'Tabs',
                      params: {screen: 'Home'},
                    },
                    {
                      name: 'Wallet',
                      params: {screen: 'KeyOverview', params: {id: key.id}},
                    },
                  ],
                }),
              );```

**Review by rastajpa — CHANGES_REQUESTED** [review]

File: src/navigation/wallet/screens/AddWallet.tsx (hunk 1 above)
Comment (cmgustavo) [reply src/navigation/wallet/screens/AddWallet.tsx:501]: +1 Thanks!

**Review by cnguyen812 — APPROVED** [review]
lgtm

File: src/navigation/wallet/screens/AddWallet.tsx (hunk 1 above)
Comment (cmgustavo) [reply src/navigation/wallet/screens/AddWallet.tsx:502]: It doesn't work. Maybe commonActions requires more params.

Comment (cmgustavo) [reply src/navigation/wallet/screens/AddWallet.tsx:502]: Just added navigation.dispatch 👍🏼 
Thanks!

**Review by cnguyen812 — APPROVED** [review]

**Review by JohnathanWhite — APPROVED** [review]

**Pull request merged**